### PR TITLE
sort inner content

### PR DESCRIPTION
**Pull request by vcarmen**
Add a sort inner content class and unit test

### CODE REVIEW

File: src/test/java/org/fundacionjala/coding/carmen/SortContentTest.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package org.fundacionjala.coding.carmen;
+
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+
+public class SortContentTest {
```
Comment (carledriss) [inline src/test/java/org/fundacionjala/coding/carmen/SortContentTest.java:6]: Your implementation is failing with this scenario 
```java
assertEquals("I am", SortContent.sortTheInnerContent("I am"));
```

File: src/main/java/org/fundacionjala/coding/carmen/SortContent.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+            int wordSize = sentence[i].length();
+            String innerString = sentence[i].substring(1, wordSize - 1);
+            String firstChar = sentence[i].substring(0, 1);
+            String lastChar = sentence[i].substring(wordSize - 1, wordSize);
+
+            char[] arr = innerString.toCharArray();
+            Arrays.sort(arr);
+            innerString = new StringBuilder(String.valueOf(arr)).reverse().toString();
+
+            sentence[i] = firstChar + innerString + lastChar;
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/coding/carmen/SortContent.java]: You can take advantage of Arrays.sort to sort descending 
```java
            String[] innerArray = innerString.split("");
            Arrays.sort(innerArray, Comparator.reverseOrder());
            sentence[i] = firstChar + String.join("", innerArray) + lastChar;
```

**Review by carledriss — CHANGES_REQUESTED** [review]

File: src/test/java/org/fundacionjala/coding/carmen/SortContentTest.java (hunk 1 above)
Comment (vcarmen) [reply src/test/java/org/fundacionjala/coding/carmen/SortContentTest.java:6]: thanks, observation resolved

File: src/main/java/org/fundacionjala/coding/carmen/SortContent.java (hunk 2 above)
Comment (vcarmen) [reply src/main/java/org/fundacionjala/coding/carmen/SortContent.java]: thanks, observation resolved

**Review by carledriss — APPROVED** [review]

**Pull request merged**